web/packages/teleport/src/DeviceTrust/types.ts

```diff
@@ -41,9 +41,3 @@ export type DeviceListProps = {
   fetchStatus?: 'loading' | 'disabled' | '';
   fetchData?: () => void;
 };
-
-export enum TrustedDeviceRequirement {
```

Comment:
Was the enum duplicated here before the proto changes? I honestly don't mind this.

Reply:
Yes, the Web UI had its own declaration of this enum. I found it by accident, and thought that it's better to use the proto type (which has proper docs).

If you are concerned about proto imports in Web UI, it's not a new thing; we already import types/enums in other parts of it.

Reply:
Thanks for the explanation.